papaya-core/src/main/java/pl/edu/agh/papaya/model/LocalDateTimePeriod.java

```diff
@@ -65,6 +65,14 @@ public void extendToContain(LocalDateTime date) {
         }
     }
 
+    public boolean isAfter(LocalDateTimePeriod period) {
```

Comment:
`isAfter` i `isBefore` to chyba trochę nietrafne nazwy, ja np. `isAfter(period)` rozumiem jako:
```java
this.start.isAfter(period.getEnd())
```

Reply:
i masz racje bo to bug